### PR TITLE
Fix checkstyle

### PR DESCRIPTION
**Pull request by Kafcis**
I went through the code to fix the checkstyle issues

### CODE REVIEW

File: src/main/java/athena/Storage.java [hunk 1]
```diff
@@ -67,23 +44,30 @@ public void saveTaskListData(TaskList tasks) {
...
             try {
                 csvReader = new BufferedReader(new FileReader(filePath));
                 while ((row = csvReader.readLine()) != null) {
                     String[] data = row.split(",");
-                    output.addTask(data[0],data[1],data[2],data[3],data[4],data[5],data[6]);
+                    output.addTask(data[0], data[1], data[2], data[3], data[4], data[5], data[6],
+                            Integer.parseInt(data[7]));
+                    maxIndex = Integer.parseInt(data[7]);
                 }
+
```
Comment (alstontham) [inline src/main/java/athena/Storage.java]: would checkstyle flag this extra space?

File: src/main/java/athena/Storage.java [hunk 2]
```diff
@@ -67,23 +44,30 @@ public void saveTaskListData(TaskList tasks) {
...
                 csvReader.close();
             } catch (FileNotFoundException e) {
                 e.printStackTrace();
             } catch (IOException e) {
                 e.printStackTrace();
+            } catch (ArrayIndexOutOfBoundsException e) {
+                ui.printInvalidTask();
             }
```
Comment (alstontham) [inline src/main/java/athena/Storage.java]: would checkstyle flag this extra space?

**Review by alstontham — COMMENTED** [review]
Looks good! Just a few extra empty lines that the checkstyle could possibly flag.

**Review by daniellimws — APPROVED** [review]
Since checkstyle approves I think this should be good. 

But we should only merge this after the other PRs are merged because some of the things here might have not been approved in the other PRs.

**Comment by daniellimws** [comment]
Er this PR has error with checkstyle test after you merged master into it. We shouldn't have merged.

**Pull request merged**